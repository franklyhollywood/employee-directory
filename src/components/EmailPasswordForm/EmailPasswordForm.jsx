import React from 'react';

export default function EmailPasswordForm(props) {
  const text = props.signin ? 'SIGN IN' : 'SIGN UP';
  return (
    <div>
      <h2>Welcome!</h2>
      <form>
        <fieldset>
          <legend>{text}</legend>
          <div>
            <label>Email</label>
            <input type="email" />
          </div>
          <div>
            <label>Password</label>
            <input type="password" />
          </div>
          <button>{text}</button>
        </fieldset>
      </form>
    </div>
  );
}
