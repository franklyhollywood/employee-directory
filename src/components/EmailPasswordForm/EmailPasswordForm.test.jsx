import { MemoryRouter, Switch, Route } from 'react-router-dom';
import { render } from '@testing-library/react';
import EmailPasswordForm from './EmailPasswordForm.jsx';

it('should render form', () => {
  const { container } = render(<EmailPasswordForm />);

  expect(container).toMatchSnapshot();
});
