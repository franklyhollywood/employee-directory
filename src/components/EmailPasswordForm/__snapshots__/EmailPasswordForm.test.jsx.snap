// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render form 1`] = `
<div>
  <div>
    <form>
      <fieldset>
        <legend>
          SIGN UP
        </legend>
        <div>
          <label
            for="email"
          >
            Email
          </label>
          <input
            id="email"
            type="email"
            value=""
          />
        </div>
        <div>
          <label
            for="password"
          >
            Password
          </label>
          <input
            id="password"
            type="password"
            value=""
          />
        </div>
        <button
          type="submit"
        >
          SIGN UP
        </button>
      </fieldset>
    </form>
  </div>
</div>
`;
